editor/src/components/canvas/controls/insertion-plus-button.spec.browser2.tsx

```diff
@@ -305,9 +305,9 @@ describe('Insertion Plus Button', () => {
       expect(bounds.top).toEqual(expectedTop)
     }
 
-    await checkInsertButtonPosition('blue-dot-control-0', 458.5, 631.5)
+    await checkInsertButtonPosition('blue-dot-control-0', 785.5, 631.5)
     await checkInsertButtonPosition('blue-dot-control-1', 651.5, 631.5)
-    await checkInsertButtonPosition('blue-dot-control-2', 785.5, 631.5)
+    await checkInsertButtonPosition('blue-dot-control-2', 458.5, 631.5)
   })
```

Comment:
this is deliberate, I just changed the order of these buttons, on master they always go LTR, on this branch, they follow the order of the actual elements.